PostHog/Classes/PHGPostHog.m

```diff
@@ -281,32 +281,53 @@ - (void)group:(NSString *_Nonnull)groupType groupKey:(NSString *_Nonnull)groupKe
 }
 
 - (id)getFeatureFlag:(NSString *)flagKey
+{
+    return [self getFeatureFlag:flagKey options:nil];
+}
+
+- (id)getFeatureFlag:(NSString *)flagKey options:(NSDictionary *)options
 {
     NSDictionary *variants = [self.payloadManager getFlagVariants];
     id variantValue = [variants valueForKey:flagKey];
-    NSMutableDictionary *properties = [NSMutableDictionary dictionary];
-    [properties setValue:flagKey forKey:@"$feature_flag"];
-    [properties setValue:variantValue forKey:@"$feature_flag_response"];
-    
-    [self run:PHGEventTypeCapture payload:
-                                    [[PHGCapturePayload alloc] initWithEvent:@"$feature_flag_called"
-                                                                  properties:PHGCoerceDictionary(properties)]];
+
+    id send_event = [options valueForKey:@"send_event"];
+
+    if (send_event == nil || [send_event boolValue] != false) {
+        NSMutableDictionary *properties = [NSMutableDictionary dictionary];
+        [properties setValue:flagKey forKey:@"$feature_flag"];
+        [properties setValue:variantValue forKey:@"$feature_flag_response"];
+        
+        
+        [self run:PHGEventTypeCapture payload:
+                                        [[PHGCapturePayload alloc] initWithEvent:@"$feature_flag_called"
+                                                                    properties:PHGCoerceDictionary(properties)]];
+    }
+
     return variantValue;
 }
 
 - (bool)isFeatureEnabled:(NSString *)flagKey
+{
+    return [self isFeatureEnabled:flagKey options:nil];
+}
+
+- (bool)isFeatureEnabled:(NSString *)flagKey options:(NSDictionary *)options
```

Comment:
Nit, I think both these functions for `isFeatureEnabled` should start using `getFeatureFlag` now!

Reply:
true, not going to handle at the moment in this PR